Components/typography/typography.html

```diff
@@ -28,14 +28,15 @@
             <div class="nav__right">
                 <div class="social-media-list">
                     <ul class="social-media-list">
-                        <li class="social-media-items"><a href="#"><img class="icon-height"
-                                    src="../../images/Linkedin.png" alt="Linkedin"></a></li>
-                        <li class="social-media-items"><a href="#"><img class="icon-height"
-                                    src="../../images/github.png" alt="github"></a>
-                        </li>
-                        <li class="social-media-items"><a href="#"><img class="icon-height"
-                                    src="../../images/twitter.png" alt="twitter"></a>
-                        </li>
+                        <li class="social-media-items"><a target="_blank"
```

Comment:
It can be seen that 2 patterns are used for class names. Isn't it better to stick to one?